server/coordinator/procedure/scatter.go

```diff
@@ -69,20 +71,31 @@ func scatterPrepareCallback(event *fsm.Event) {
 		return
 	}
 
+	shardTopologies := make([]*clusterpb.ShardTopology, 0)
 	for _, shard := range shards {
 		openShardRequest := &eventdispatch.OpenShardRequest{
 			Shard: &cluster.ShardInfo{
 				ID:   shard.GetId(),
 				Role: clusterpb.ShardRole_LEADER,
 			},
 		}
+		shardTopologies = append(shardTopologies, &clusterpb.ShardTopology{
+			ShardId:  shard.GetId(),
+			TableIds: make([]uint64, 0),
```

Comment:
For empty slice, it will be more simple and concise to use `[]int64{}` to do such initialization:
```suggestion
			TableIds: []uint64{}
```